tests/test_config.py

```
@@ -75,3 +83,177 @@ def test_transport_config_invalid_port(port) -> None:
+        client_id="id",
+        client_secret="secret",
+        zammad_base_url="https://z.example.com",
+    )
+    with pytest.raises(ValueError, match="MCP_AUTH_BASE_URL"):
+        config.validate()
+
+
+def test_auth_config_validate_multiple_missing() -> None:
+    """Test validation reports all missing fields."""
+    config = AuthConfig(client_id="id")
+    with pytest.raises(ValueError, match="MCP_AUTH_CLIENT_SECRET.*ZAMMAD_URL.*MCP_AUTH_BASE_URL"):
```

Ruff (0.15.12)

[warning] Pattern passed to `match=` contains metacharacters but is neither escaped nor raw

(RUF043)